Analysing ../../SCRUBD/SCRUBD-CD/data/solidity_codes/0x2268003d92ff03be6ca6da029f8d8127fd2f617e.sol:SafeMath...
Analysing ../../SCRUBD/SCRUBD-CD/data/solidity_codes/0x2268003d92ff03be6ca6da029f8d8127fd2f617e.sol:ShortOrder...
Analysing ../../SCRUBD/SCRUBD-CD/data/solidity_codes/0x2268003d92ff03be6ca6da029f8d8127fd2f617e.sol:Token...

--- STDERR ---

Traceback (most recent call last):
  File "/home/sh1kaku/lihao/conkas/conkas.py", line 105, in main
    ssa = Recover(bytecode, edges=[], optimize=True)
  File "/home/sh1kaku/lihao/conkas/rattle/recover.py", line 768, in __init__
    self.internal = InternalRecover(filedata, edges, optimize, split_functions)
  File "/home/sh1kaku/lihao/conkas/rattle/recover.py", line 37, in __init__
    self.guarenteed_optimizations()
  File "/home/sh1kaku/lihao/conkas/rattle/recover.py", line 674, in guarenteed_optimizations
    if reader.insn.is_push and reader.replace_uses_with(item):
  File "/home/sh1kaku/lihao/conkas/rattle/ssa.py", line 397, in replace_uses_with
    reader.replace_argument(self.return_value, sv)
  File "/home/sh1kaku/lihao/conkas/rattle/ssa.py", line 289, in replace_argument
    assert (self.insn.pops == len(self.arguments) or
AssertionError
